docs/docs/installation/docker.mdx

```diff
@@ -0,0 +1,51 @@
+# Docker Images and Tags
+
+The Apache Superset community extensively uses Docker for development, release,
+and productionizing Superset. This page details our Docker builds and tag naming
+schemes to help users navigate our offerings.
+
+Images are built and pushed to the [Superset Docker Hub repository](
+https://hub.docker.com/r/apache/superset). Different sets of images are created for:
+- Published releases, with tags like `3.0.0` and the `latest` tag.
+- Pull request iterations, each identified by tags starting with a SHA like
+  `8a2f7d378ab13c156fa183d9284b607ed69f5ecc`, and `pr-3454`, referencing the pull
+  request ID.
+- Merges to the main branch (`master`), resulting in new SHAs, with tags
+  prefixed with `master` for the latest `master` version.
+
+Each code version has multiple builds for different purposes, identified by suffixes:
+- **Build Preset:** We offer various images for different needs:
```

Comment:
Does "Preset" here refer to the company?  Or does it mean "prefix" or something else?  If it's not referring to Preset the company, it may be clearer to use another term.

Reply:
Unrelated to the company. I really mean it its it's dictionary definition 
<img width="851" alt="Screenshot 2024-02-01 at 12 06 51 PM" src="https://github.com/apache/superset/assets/487433/da884e0e-d0f6-43e3-9399-41c4956ec3f2">